lib/block-supports/colors.php

```diff
@@ -60,6 +60,14 @@ function gutenberg_register_colors_support( $block_type ) {
  */
 function gutenberg_apply_colors_support( $block_type, $block_attributes ) {
 	$color_support                 = gutenberg_experimental_get( $block_type->supports, array( 'color' ), false );
+
+	if ( array_key_exists( 'skipSerialization', $color_support ) && $color_support['skipSerialization'] ) {
+		return array(
```

Comment:
We can probably return an empty array here, don't you think?

Reply:
Oh, yes, it doesn't make sense as it was. Fixed in https://github.com/WordPress/gutenberg/pull/29142/commits/29d4578a19da67adebd26be74aa70230daabe030